### ISSUE TITLE
[FEAT-1251] build:env:create WordPress multisite support for initial environment creation

### ISSUE THREAD

**Issue by carl-alberto** [pantheon-systems/terminus-build-tools-plugin#475]
At the moment, the build tools plugin only supports wp but not the wp multisite framework

tried to do the `terminus site:upstream:set` command to see if it was a possible workaround but failed

![Screen Shot 2023-03-31 at 9 15 24 PM](https://user-images.githubusercontent.com/5098765/229130282-fb812d9f-7339-468f-9ab8-98c2c79fe868.png)

**Comment by j-masci** [comment]
@carl-alberto Were you able to figure out anything regarding this? Did you have to avoid using Build Tools with your multisite framework environment?

**Comment by pwtyler** [comment]
Tracking this internally as FEAT-1251.

**Comment by carl-alberto** [comment]
> @carl-alberto Were you able to figure out anything regarding this? Did you have to avoid using Build Tools with your multisite framework environment?

yes, while there are no native WPMS support I had to skip using Build Tools unfortunately
